src/Microsoft.Diagnostics.Monitoring.WebApi/IEgressOutputOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi
+{
+    public enum EgressMode
+    {
+        HttpDisabled,
+        HttpEnabled
+    }
+
+    public interface IEgressOutputOptions
+    {
+        EgressMode EgressMode { get; }
```

Comment:
This could be simplied to `bool IsHttpEgressEnabled` and then the enum can be removed. I don't know if we'll really have any other values on the enum other than enabled and disabled, so at that point its just replicating the behavior of a Boolean.